tests/integration/fundRanking.js

```diff
@@ -0,0 +1,125 @@
+import test from "ava";
+import api from "../../utils/lib/api";
+import {deployContract} from "../../utils/lib/contracts";
+import deployEnvironment from "../../utils/deploy/contracts";
+import getSignatureParameters from "../../utils/lib/getSignatureParameters";
+
+const environmentConfig = require("../../utils/config/environment.js");
+
+const environment = "development";
+const config = environmentConfig[environment];
+
+let accounts
+let fundRanking;
+let deployed;
+let version;
+let manager;
+let deployer;
+
+const fundNames = ["Fund Name 1", "Fund Name 2"];
+
+// To convert bytes to string
+function binToString(array){
+	let result = "";
+  const bytes = array._value;
+	for(let i = 0; i < bytes.length; i += 1){
+    const convertedChar = (String.fromCharCode(bytes[i]));
+    if (convertedChar === "\u0000") {
+      break;
+    }
+		result+= convertedChar;
+	}
+	return result;
+}
+
+// To return values from the array
```

Comment:
Not quite sure what this function is doing.
Since it is only called once, maybe we can get rid of it and just destructure the result of `getFundDetails` inside the unit test itself.
What do you think?